Utility.cpp

```diff
@@ -31,78 +30,79 @@ std::vector<std::unique_ptr<Item>> makeDefensiveItems(int num)
     return items;
 }
 
-std::string getCharacterStats(Character* ch)
+std::string getCharacterStats(Character* charName)
 {
     std::string str;
     
-    str += "    hitPoints: " + std::to_string(ch->getHP()) + "\n";
-    str += "    armor: " + std::to_string(ch->getArmorLevel()) + "\n";
-    str += "    attack damage: " + std::to_string(ch->getAttackDamage()) + "\n";
-    str += "    is defending: " + std::string((ch->getIsDefending() ? "true" : "false" )) + "\n";
-    str += "    " + std::to_string(ch->getHelpfulItems().size()) + " helpful items,  " + std::to_string(ch->getDefensiveItems().size()) + " defensive items";
+    str += "    hitPoints: " + std::to_string(charName->getHP()) + "\n";
+    str += "    armor: " + std::to_string(charName->getArmorLevel()) + "\n";
+    str += "    attack damage: " + std::to_string(charName->getAttackDamage()) + "\n";
+    str += "    is defending: " + std::string((charName->getIsDefending() ? "true" : "false" )) + "\n";
+    str += "    " + std::to_string(charName->getHelpfulItems().size()) + " helpful items,  " + std::to_string(charName->getDefensiveItems().size()) + " defensive items";
```

Comment:
This did not need revising.
You need to learn to not change code that doesn't need to be changed.
- [ ] revert